host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -323,7 +323,9 @@ static COMMAND_LIST cmdline_commands[] =
    { CommandWidth,         "-width",      "w",  "Set image width <size>. Default 1920", 1 },
    { CommandHeight,        "-height",     "h",  "Set image height <size>. Default 1080", 1 },
    { CommandBitrate,       "-bitrate",    "b",  "Set bitrate. Use bits per second (e.g. 10MBits/s would be -b 10000000)", 1 },
-   { CommandOutput,        "-output",     "o",  "Output filename <filename> (to write to stdout, use '-o -')", 1 },
+   { CommandOutput,        "-output",     "o",  "Output filename <filename> (to write to stdout, use '-o -').\n"
+         "\t\t  Connect to a remote Host (e.g. tcp://192.168.1.2:1234, udp://192.168.1.2:1234)\n"
+         "\t\t  Listen on a TCP port and wait for an incoming connection (e.g. ltcp://0.0.0.0:1234, ltcp://192.168.1.1:1234)", 1 },
```

Comment:
LTCP seems like an unfortunate choice, since it is an existing (albeit draft, and now forgotten) protocol:

https://tools.ietf.org/html/draft-bhandarkar-ltcp-00

A separate option to enable this might (as others have said) be better.